Source/WebCore/page/Navigator.cpp

```diff
@@ -265,7 +265,7 @@ void Navigator::initializePluginAndMimeTypeArrays()
     });
 
     std::sort(webVisibleDOMMimeTypes.begin(), webVisibleDOMMimeTypes.end(), [](const Ref<DOMMimeType>& a, const Ref<DOMMimeType>& b) {
-        if (auto typeComparison = codePointCompare(a->type(), b->type()))
+        if (auto typeComparison = codePointCompare(a->type().string(), b->type().string()))
```

Comment:
Same comment about overloading. Can we overcome the ambiguity in the headers instead of touching call sites like this one?

Reply:
Will fix.